tests/core/test_atexit_cxa.c

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Sorry can you rename this back to test_atexit_tread?   I'm sorry I misunderstood what `__cxa_thread_atexit` was for.

With that lgtm.